torch/export/_trace.py

```diff
@@ -368,6 +368,13 @@ def _export_non_strict(
     if isinstance(mod, torch.fx.GraphModule) and hasattr(mod, "meta"):
         gm.meta.update(mod.meta)
 
+    if pre_dispatch:
```

Comment:
Could we add a test case for the serialization/deserialization?

Reply:
Sure! Thanks for the reminder. Can I do it as a follow-up PR? Want to keep this one short.

Reply:
Ok sounds good :)